api/tests/auth.test.ts

```diff
@@ -81,19 +81,31 @@ test('Authentication: access protected endpoint without valid token', async () =
 
 test('Authentication: token expiration error', async () => {
   await signUp(ctx.client);
-
-  vi.spyOn(env, 'JWT_EXPIRATION_PERIOD', 'get').mockReturnValue('1s');
   const signInResponse = await signIn(ctx.client);
   expect(signInResponse.signIn.token).toBeTruthy();
-  const expiry = new Date(signInResponse.signIn.session.referenceExpiryDate!).getTime();
-  const now = new Date().getTime();
-
-  const token = signInResponse?.signIn?.token;
+  setHeader({ Authorization: `Bearer ${signInResponse.signIn.token}` });
+  const createdFastExpiredSession = await ctx.client.mutation({
+    createSession: {
+      token: true,
+      session: {
+        isUserCreated: true,
+        id: true,
+        referenceExpiryDate: true,
+      },
+      __args: {
+        session: {
+          expiryDurationSeconds: 1,
+          name: 'test',
+        },
+      },
+    },
+  });
```

Comment:
As far as I understand, this is not correct use of the graphql client (hinted by the use of internal `__args` variable). Instead, you should first build the mutation in one place (eg `const createSession = ctx.client.mutation({...})`) and then use mutation in your test `await createSession({ name: 'test', expiryDurationSeconds: 1)` otherwise you need to repeat a lot of unnecessary parts of the code. But I can't find the documentation of `client` you're using since it's completely generated and uses `fetch` under the hood. Can you please point to the documentation based on which this mutation is written? Or can you link to the docs on the lib that generates the client?